### PR TITLE
add firefox support

### PR DESCRIPTION
**Pull request by artemgurzhii**
**Problem:** in the [search flow](https://github.com/alechirsch/ember-search-flow/blob/master/addon/components/search-flow.js#L143) component you are using global window `event` property, which returns the Event which is currently being handled by the site's code. 

According to the [window event browser support](https://developer.mozilla.org/en-US/docs/Web/API/Window/event#Browser_Compatibility#Browser_Compatibility) it's being supported in all browsers and firefox since version 63.

According to the [ember action events doc](https://guides.emberjs.com/release/components/handling-events/#toc_sending-actions) event will be passed to the action, only when assigning the action to an inline handler.

So basically this PR removes usage of the `window.event` and replaces it with the event argument itself.

**P.S.** Tested on the Firefox#62 and Chrome#69

**P.P.S** Couldn't run tests because of [this error.](https://monosnap.com/direct/xRilLfZJk2HaK0ALlc3aRiutahDpoy)

### CODE REVIEW

**Comment by alechirsch** [comment]
@artemgurzhii Thank you for submitting this, I will also look into deprecation notices

**Comment by artemgurzhii** [comment]
@alechirsch You're welcome, could please also add new release?

**Comment by alechirsch** [comment]
@artemgurzhii As soon as I update the deprecations I will put out a release

**Comment by artemgurzhii** [comment]
@alechirsch Great, also I have thought about adding fastboot support, what do you think about it?

**Comment by alechirsch** [comment]
@artemgurzhii I have never used it, I will welcome a PR though

**Comment by alechirsch** [comment]
@artemgurzhii 0.6.4 has been published

**Comment by alechirsch** [comment]
Thanks for using the add on, I hope you like it
